src/io/flutter/run/test/TestConfigProducer.java

```diff
@@ -47,6 +48,8 @@ private boolean setupForDartFile(TestConfig config, ConfigurationContext context
     final PubRoot root = PubRoot.forPsiFile(file);
     if (root == null) return false;
 
+    if (!FlutterModuleUtils.hasFlutterModule(file.getProject())) return false;
```

Comment:
Do we need to do a similar check in `setupForDirectory`?

Thanks for the fix! I would get Brian's eyes on this. If this style of check looks good, I would use the module available on the `ConfigurationContext`, and call the `FlutterModuleUtils .usesFlutter` w/ that module (a bit more scoped than the project).